app/controllers/collection_drafts_controller.rb

```diff
@@ -116,8 +89,10 @@ def update
       end
     else # record update failed
       # render 'edit' # this should get get_resource_form
-      # Remove
       flash[:error] = I18n.t("controllers.draft.#{plural_resource_name}.update.flash.error")
```

Comment:
We've discussed this, but now it seems that the template specific error will not go in the flash, the flash message will be a generic message. But because there is a save error for the model object, we that error will still be presented to the user. That seems like a good solution.